fetch-in-react/src/app.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import UserList from './user-list';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      isLoading: true
+    };
+  }
+
+  componentDidMount() {
+    fetch('https://jsonplaceholder.typicode.com/users')
+      .then(response => response.json())
+      .then(data => {
+        const userList = [];
+        for (let i = 0; i < 10; i++) {
+          userList.push(data[i]);
+        }
```

Comment:
The `data` is already inside an array so there is no need to loop through it and push it into another array. Here you can just do:
```JAVASCRIPT
this.setState({ users: data, isLoading: false });
```